.buildkite/pipeline.yml

```diff
@@ -35,7 +37,8 @@ steps:
    commands: &run-nettest
      - nix build -L -f. nettest
      - mkdir -p test/resources/
-     - nix-build -A tezos-contract -o test/resources/stablecoin.tz
+     - nix-build -A tezos-contract -o stablecoin.tz
+     - nix run -f. morley -c morley optimize --contract stablecoin.tz --output test/resources/stablecoin.tz
```

Comment:
Now I see some non-trivial amount of duplication between this step and the one where we run `stablecoin-test`. Is it possible to deduplicate common parts?